resources/js/views/general/Upload.vue

```diff
@@ -69,8 +75,19 @@ export default {
         verror (file)
         {
             console.log({ file });
-            // this.error = true
-            // window.toastr.error(file.upload.filename, 'Event : vdropzone-error - ' + file.status)
+
+            let errorMessage = 'Something went wrong. Please try again later.';
```

Comment:
We need to be more descriptive and it should be translated

When an image does not have a geotag, the error currently says "Sorry, no GPS tag found". This error would overwrite that?

Reply:
Yes, this error just in case we got some unknown error.